PluginCode/src/main/java/smr/cs/ualberta/libcomp/DatabaseAccess.java

```diff
@@ -394,7 +397,7 @@ public boolean sendFeedback(String urlStr, String jsonBodyStr, String tokenValue
         }
 
         try {
-            if (httpURLConnection.getResponseCode() == HttpURLConnection.HTTP_OK) {
+            if (httpURLConnection.getResponseCode() == 201) {
```

Comment:
Why this change? Is HTTP_OK != 201? Using the named value is more readable

Reply:
ok, now using the named variables